Formula/gtk+3.rb

```diff
@@ -25,7 +26,7 @@ class Gtkx3 < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on "atk"
+  depends_on "at-spi2-core"
```

Comment:
This won't work as `at-spi2-core` is Linux-only and we will lose all macOS bottles and break GTK+3 dependents.